lib/app.js

```diff
@@ -30,7 +30,7 @@ app.get('/api/test', (req, res) => {
 
 app.get('/animals', async(req, res) => {
   try {
-    const data = await client.query('SELECT * from animals');
+    const data = await client.query('SELECT * from chords');
```

Comment:
Great job with this `SELECT` query! You just need to update your endpoint URL on line 31 to match your new dataset instead of the original `/animals`.